app/search/dataone.py

```diff
@@ -103,6 +103,9 @@ def date_filter_search(self, **kwargs):
 
     def combined_search(self, **kwargs):
         text = kwargs.pop('text', None)
```

Comment:
Is there a `text_search` method in here? if so, you'll probably want to do the same thing in it.

Reply:
Oh yes there is, i would add it there thanks!